src/backend/session/mod.rs

```diff
@@ -70,39 +70,15 @@ pub trait Session {
     fn seat(&self) -> String;
 }
 
-/// Signals that can be generated by a session
-///
-/// Objects that need to be notifier about activation and deactivation
-/// of devices need to be linked to the `Signaler` of the session providing
-/// these signals.
+/// Events that can be generated by a session
 #[derive(Copy, Clone, Debug)]
 pub enum Signal {
```

Comment:
Kinda nitpicky, but I wonder if the name Signal / SessionSignal, is still a good choice since it is no longer a calloop signal, and just a regular event.

Reply:
Fixed